### PR TITLE
[formal/conn] Support connectivity test locally

### PR DESCRIPTION
**Pull request by cindychip**
This PR adds basic configs to run connecivity test locally.
This is the first step to support running connecivity test using dvsim.
The reason we are adding this local option is because we can temporarily
using this script to run with GUI.

Next PR will support running the connectivity test on dvsim.

Signed-off-by: Cindy Chen <chencindy@google.com>

### CODE REVIEW

File: hw/formal/formal_conn [hunk 1]
```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```
Comment (cindychip) [inline hw/formal/formal_conn:1]: This batch script is temp support for user to run connectivity test locally with GUI. Once add the -GUI support in dvsim, we can remove this script.

Comment (weicaiyang) [inline hw/formal/formal_conn:1]: we usually name bash script with `.sh`

**Review by weicaiyang — APPROVED** [review]
LGTM with a suggestion

File: hw/formal/formal_conn (hunk 1 above)
Comment (cindychip) [reply hw/formal/formal_conn:1]: Thanks Weicai for the reminder, let me update the naming :)

File: hw/formal/formal_conn.sh [hunk 2]
```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```
Comment (cindychip) [inline hw/formal/formal_conn.sh:1]: This batch script is temp support for user to run connectivity test locally with GUI. Once add the -GUI support in dvsim, we can remove this script.

File: hw/formal/conn_csvs/top_earlgrey_conn.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+,NAME,SRC BLOCK,SRC SIGNAL,DEST BLOCK,DEST SIGNAL,,,,,,
+
+CONNECTION,CLKMGR_IDLE0,u_clkmgr_aon,idle_i[0],u_aes,idle_o
```
Comment (cindychip) [inline hw/formal/conn_csvs/top_earlgrey_conn.csv:3]: Hi @matutem, this is just a trial from clock manager, feel free to expand or remove it.

**Review by udinator — APPROVED** [review]

**Comment by cindychip** [comment]
Thanks everyone for reviewing, close this one as reviews are in #5770

**Pull request closed without merging**